wormhole-connect/src/components/Header.tsx

```diff
@@ -4,19 +4,17 @@ import { makeStyles } from 'tss-react/mui';
 export type Alignment = 'center' | 'left' | 'right';
 
 type StyleProps = { align: Alignment; fontSize: number };
-const useStyles = makeStyles<StyleProps>()(
```

Comment:
I have no idea why but this code was reformatted and I can't revert it because git insists the commit would be empty (because of the precommit hook)